recipes.js

```diff
@@ -0,0 +1,67 @@
+const recipes = require("express").Router()
+let allRecipes = [{
+    name:"Grilled Cheese",
+    ingredients:["bread", "butter", "cheese"],
+    directions: "Preheat skillet over medium heat. Generously butter one side of a slice of bread. Place bread butter-side-down onto skillet bottom and add 1 slice of cheese. Butter a second slice of bread on one side and place butter-side-up on top of sandwich. Grill until lightly browned and flip over; continue grilling until cheese is melted. Repeat with remaining 2 slices of bread, butter and slice of cheese."
+},
+{
+    name:"Peanut Butter and Jelly",
+    ingredients:["bread", "Peanut Butter", "Jelly"],
+    directions: "Spread the peanut butter on one piece of bread.Spread the jelly on the other side. Put the two pieces of bread together to form a sandwich. Toddler adaptation: cut off crusts before serving"
+},
+{
+    name:"Buttered Toast",
+    ingredients:["bread", "butter"],
+    directions: "Generously butter one side of a slice of bread."
+}]
+recipes.get("/", (req, res)=>{
+
+    res.json(allRecipes)
+})
+
+recipes.post("/newRecipes", (req, res)=>{
```

Comment:
I recommend using req.body as a way to get post requests information

Reply:
how you would do this would be by having the object you want to send back as the second argument on the axios post request.

`axios.post('url', {object you want to send to backend to be reached by req.body})`